src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlTime.java

```diff
@@ -50,7 +49,18 @@ public SqlTime() {
 
     public Object createdata() {
         Time temp = new Time(ThreadLocalRandom.current().nextLong(((Time) minvalue).getTime(), ((Time) maxvalue).getTime()));
-        String timeNano = temp.toString() + "." + RandomStringUtils.randomNumeric(this.scale);
+        Random rnd = new Random();
+        // this will generate a number that is this.scale number of digits.
+        int randomNumeric;
+        if (this.scale <= 0) {
+            randomNumeric = 0;
+        } else if (this.scale == 1) {
+            randomNumeric = rnd.nextInt(10);
+        } else {
+            randomNumeric = (int) Math.pow(10, this.scale - 1) + rnd.nextInt((int) (9 * Math.pow(10, this.scale - 1)));
+        }
+        
```

Comment:
this is the same as above can we move this out to a method which generates n random digits?